pmoves/services/audio-reprocess/app.py

```
@@ -0,0 +1,242 @@
+@app.get("/healthz")
+async def healthz():
+    return {"status": "ok", "device": DEVICE, "model": WHISPER_MODEL_NAME}
+
+
+@app.post("/api/v1/audio/reprocess")
+async def reprocess(req: ReprocessRequest):
+    try:
+        return await run_pipeline(req.audio_url)
+    except Exception as exc:
+        LOG.exception("reprocess failed for %s", req.audio_url)
+        raise HTTPException(status_code=500, detail=str(exc))
```

Ruff (0.15.20)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)